plugins/com.google.cloud.tools.eclipse.appengine.facets.test/src/com/google/cloud/tools/eclipse/appengine/facets/ui/navigator/ConfigurationFileUtils.java

```diff
@@ -69,9 +67,7 @@ private static IFile createFileInWebInf(IProject project, IPath path, String con
           new ByteArrayInputStream(contents.getBytes(StandardCharsets.UTF_8));
       return WebProjectUtil.createFileInWebInf(project, path, stream, true, null);
     } catch (CoreException ex) {
-      fail(ex.toString());
-      /*NOTREACHED*/
-      return null;
+      throw new AssertionError(ex.toString(), ex);
```

Comment:
We probably shouldn't be throwing errors in non-test code.

Reply:
This is a test utility